packages/frontend/src/components/Game/GameBoard/index.test.jsx

```diff
@@ -1,5 +1,80 @@
+import { setupTests } from 'helpers/tests';
 import GameBoard from '.';
+import { mockPhotos } from '__mocks__/api/mockPhotos';
+import { createTilesFromPhotos } from 'helpers/createTilesFromPhotos';
+import { baseState } from 'contexts';
+import { produce } from 'immer';
+import { act } from '@testing-library/react';
+import { byTestId, byAltText } from 'testing-library-selector';
+
+beforeEach(() => {
+  jest.useFakeTimers('legacy');
+});
+
+afterEach(() => {
+  jest.useRealTimers();
+});
+
+jest.spyOn(global, 'setTimeout');
 
 describe('GameBoard component', () => {
-  it('', () => {});
+  it('flips the tile thats been clicked', async () => {
+    const tiles = createTilesFromPhotos(mockPhotos, { shuffle: false });
+
+    const initialGameState = produce(baseState.game, (draft) => {
+      draft.tiles = tiles;
+    });
+    const state = { ...baseState, game: initialGameState };
+
+    const { screen, user } = setupTests(GameBoard, { state });
+
+    const element = screen.getByTestId('tile-1');
+
+    expect(screen.getByTestId('tile-1')).toHaveClass('faceDown');
+    await user.click(element);
+
+    jest.advanceTimersByTime(3000);
+
+    const displayedImage = document.querySelector('img');
+
+    expect(screen.getByTestId('tile-1')).not.toHaveClass('faceDown');
+    expect(screen.getByAltText(/Jellyfish/)).toBeInTheDocument();
+    expect(displayedImage).toBeInTheDocument();
```

Comment:
I'm not sure this assertion needs to exist. The test on line 41 serves the same purpose because it should fail if the image isn't rendered in the document.